source/_docs/free-https.md

```diff
@@ -0,0 +1,205 @@
+      <pre><code>terminus remote:drush &lt;site&gt;.&lt;env&gt; -- pm-enable hsts --yes</code></pre>
+      </li>
+      <li>Visit the module configuration page (<code>/admin/config/security/hsts</code>).</li>
+      <li>Check the <strong>Enable HTTP Strict Transport Security</strong> checkbox, set <strong>Max Age</strong> to <code>15552000</code> and click <strong>Save Configuration</strong>.</li>
+    </ol>
+    <p>Once installed and configured, the following header will be sent in responses:</p>
+    <pre><code>strict-transport-security: max-age=15552000</code></pre>
+  </div>
+  <div role="tabpanel" class="tab-pane" id="drops8">
+    <ol>
+      <li>Install the <a href="https://drupal.org/project/hsts">HTTP Strict Transport Security</a> module using the <a href="https://www.drupal.org/docs/8/extending-drupal-8/installing-modules">Drupal interface</a> or with <a href="/docs/terminus">Terminus</a>:
+      <pre><code>terminus remote:drush &lt;site&gt;.&lt;env&gt; -- pm-enable hsts --yes</code></pre>
+      </li>
+      <li>Visit the module configuration page (<code>/admin/config/system/hsts</code>).</li>
+      <li>Check the <strong>Enable HTTP Strict Transport Security</strong> checkbox, set <strong>Max Age</strong> to at least <strong>1 year</strong> and click <strong>Save Configuration</strong>.</li>
+    </ol>
+    <p>Once installed and configured, the following header will be sent in responses:</p>
+    <pre><code>strict-transport-security: max-age=31536000</code></pre>
+  </div>
+</div>
+
+## Compare to Legacy HTTPS
+<table class="table  table-bordered table-responsive">
+  <thead>
+    <tr>
+      <th></th>
+      <th>Legacy</th>
+      <th>Free & Automated</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+      <th>Price</th>
+      <td>$30/mo per environment</td>
+      <td>Free for all environments</td>
+    </tr>
+    <tr>
+      <th>Certificate type</th>
+      <td>Bring your own</td>
+      <td>Shared, issued by Let's Encrypt</td>
+    </tr>
+    <tr>
+      <th>Renewal</th>
+      <td>Manual</td>
+      <td>Automatic</td>
+    </tr>
+    <tr>
+      <th>Unique inbound IP Address</th>
+      <td>Yes</td>
+      <td>No</td>
+    </tr>
+    <tr>
+      <th>Client Support</th>
+      <td>94.58% of browsers<br />Some very old browsers not supported
+      <sup><a href="http://caniuse.com/#search=TLS%201.1">1</a></sup></td>
+      <td>94.33% of browsers<br />Some very old browsers not supported
+      <sup><a href="http://caniuse.com/#search=TLS%201.2">1 <a href="http://caniuse.com/#search=SNI">2</a></sup></td>
+    </tr>
+    <tr>
+      <th><a href="https://www.ssllabs.com/ssltest/">SSL Labs</a> Rating</th>
+      <td>A</td>
+      <td>A+ <a href="#http-strict-transport-security-header">with HSTS</a></td>
+    </tr>
+    <tr>
+      <th>Protocol</th>
+      <td>Vulnerable TLS 1.1</td>
```

Comment:
This should be "TLS 1.1 & 1.2" vs. "Vulnerable TLS 1.1" unless someone is aware of an inherent TLS 1.1 vulnerability.